static/src/javascripts/projects/common/views/commercial/survey/survey-adblock.html

```diff
@@ -24,33 +29,53 @@ <h3 class="survey-text__header">
         </h3>
         <div class="survey-content">
             <div class="survey-text">
-                <p class="survey-text__info">
+                <p class="survey-text__strong">
                     <%=surveyText%>
                 </p>
                 <% if (surveyTextSecond) { %>
-                <p class="survey-text__info">
-                    <%=surveyTextSecond%>
-                </p>
+                    <p class="survey-text__strong">
+                        <%=surveyTextSecond%>
+                    </p>
                 <% } %>
                 <% if (surveyTextThird) { %>
                 <p class="survey-text__info">
                     <%=surveyTextThird%>
                 </p>
                 <% } %>
+                <% if (membershipLink || signupLink) { %>
```

Comment:
Thanks. I just ran `make compile useAssetBundles=true` and it looks like it completed ok. Should I expect a failure in the compile, or a runtime problem? I assume you meant to pass the `useAssetBundle=true` to the `make compile` command?